cookbook/security/guard-authentication.rst

```diff
@@ -46,7 +46,7 @@ property they use to access their account via the API::
         /**
          * @ORM\Column(type"string", unique=true)
          */
-        private $apiKey;
+        private $apiToken;
```

Comment:
L.19 is mentioned an `apiKey` property.
IMO, `apiKey` should be kept over `apiToken`, so we should update the occurrences of `apiToken` below.